src/compiler/docs/generate-readmes.ts

```diff
@@ -116,18 +116,15 @@ async function updateReadme(config: d.Config, compilerCtx: d.CompilerCtx, readme
 
   const content: string[] = [];
 
-  const existingLines = existingContent.split(/(\r?\n)/);
```

Comment:
because here a Regexp is used delimiter is present inside the array existingLines